### PR TITLE
WRO-4920: Ignore scrollThumb and scrollbarPlaceholder when focus persist and restore

### PR DESCRIPTION
**Pull request by jeonghee27**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Focus restored to wrong element when returning panel in case focusable scrollbar

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Ignore scrollThumb and scrollbarPlaceholder when focus persist and restore.
This method has already been applied at https://github.com/enactjs/enact/pull/2994 (for moonstone)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)
I found FixedPopupPanels/WithVirtualList qa-sampler does not restore focus. So I fixed it in this PR.

### Links
[//]: # (Related issues, references)
WRO-4920

### Comments
Enact-DCO-1.0-Signed-off-by: Jeonghee Ahn (jeonghee27.ahn@lge.com)

### CODE REVIEW

File: Panels/Panel.js [hunk 1]
```diff
@@ -260,6 +260,19 @@ const PanelDecorator = hoc({defaultElement: `.${componentCss.body} *`}, (config,
 			continue5WayHold: true,
 			defaultElement,
 			enterTo: 'last-focused',
+			lastFocusedPersist: (node, all) => {
+				const filtered = all.filter(element => element && element.dataset && !element.dataset.spotlightIgnoreRestore);
```
Comment (juwonjeong) [inline Panels/Panel.js:264]: How about making the expression shorter with the optional chaining operator?
```suggestion
				const filtered = all.filter(element => element && !element.dataset?.spotlightIgnoreRestore);
```

Comment (jeonghee27) [reply Panels/Panel.js]: I fixed

**Review by juwonjeong — APPROVED** [review]
LGTM

**Review by 0x64 — APPROVED** [review]
:lgtm:

**Pull request merged**